driver/config/.schema/config.schema.json

```diff
@@ -60,6 +60,21 @@
         "hook"
       ]
     },
+    "selfServiceWebHook": {
+      "type": "object",
+      "properties": {
+        "hook": {
+          "const": "web-hook"
+        },
+        "config": {
+          "type": "object"
```

Comment:
Please provide the actual config schema here :)

Reply:
Add in https://github.com/ory/kratos/pull/1289/commits/65773bc8272f3ab75ac6c6c142e8c0a64409dde2